app/api.py

```diff
@@ -58,31 +61,35 @@ async def create(collection: str, data: Dict):
         Input Example:
         collection = "Mentees"
         data = {
-            "profile_id": "test001",
-            "first_name": "Luca",
-            "last_name": "Evans",
-            "email": "fake@email.com",
-            "city": "Ashland",
-            "state": "Oregon",
-            "country": "USA",
-            "formerly_incarcerated": true,
-            "underrepresented_group": true,
-            "low_income": true,
-            "list_convictions": [
-            "Infraction",
-            "Felony"
-            ],
-            "subject": "Web: HTML, CSS, JavaScript",
-            "experience_level": "Beginner",
-            "job_help": false,
-            "industry_knowledge": false,
-            "pair_programming": true,
-            "other_info": "Notes"
-        }
+          "profile_id": "74L22I58tn1QL4yk",
+          "mentee_intake_id": null,
+          "name": "Zayden Wilson",
+          "email": "fake@email.com",
+          "location": "New York, New York",
+          "in_US": true,
+          "formerly_incarcerated": false,
+          "underrepresented_group": false,
+          "low_income": true,
+          "convictions": "String of convictions",
+          "list_convictions": [],
+          "tech_stack": "Career Development",
+          "experience_level": "Advanced",
+          "job_help": true,
+          "industry_knowledge": false,
+          "pair_programming": false,
+          "your_hope": "String",
+          "need": "Laptop",
+          "parole_restriction": true,
+          "disability": true,
+          "work_status": true,
+          "assistance": false,
+          "other_info": "Notes"
+    }
```

Comment:
I wish the f string worked; that would have been nice, to automate changes in the database with the docstring.